notebooks/search_and_download.ipynb

```diff
@@ -8,19 +8,21 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## Installation\n",
     "\n",
-    "Python 3.8 or higher is required for this package to work. To install the package from PyPi run: `pip install satellitevu`"
+    "Python 3.8 or higher is required for this package to work. To install the package from PyPI run: `pip install satellitevu`"
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "If the package can't be installed straight off the bat, you may need to install any extra dependencies our sdk needs first. The most common is having to install the appdirs pachage (`pip install appdirs`)."
+    "If the package can't be installed straight off the bat, you may need to install any extra dependencies our SDK needs first. The most common is having to install the appdirs package (`pip install appdirs`)."
```

Comment:
This section strikes me as a little odd. Under what circumstances would we expect that Pip would either:
a) Fail to _install_ a package due to a missing runtime dependency?
b) Fail to install a dependency that's clearly indicated in pyproject.toml (and, I assume, is therefore going in the wheel)?

Reply:
This appears to be outdated now and should be removed - previously we probably didn't have `appdirs` in the `pyproject.toml` (or another reason which I can't remember) hence that line but now that package` should be installed as a dependency.